unittest/test_diskann.cpp

```diff
@@ -242,18 +246,30 @@ class DiskANNTest : public TestWithParam<std::string> {
         ASSERT_TRUE(fs::create_directory(kDir));
         ASSERT_TRUE(fs::create_directory(kIpIndexDir));
         ASSERT_TRUE(fs::create_directory(kL2IndexDir));
+        ASSERT_TRUE(fs::create_directory(kMergedL2IndexDir));
+        ASSERT_TRUE(fs::create_directory(kMergedIpIndexDir));
 
         WriteRawDataToDisk(kRawDataPath, raw_data_);
 
         knowhere::Config cfg;
+        knowhere::Config merged_cfg;
+
         knowhere::DiskANNBuildConfig::Set(cfg, build_conf);
+        knowhere::DiskANNBuildConfig::Set(merged_cfg, merged_vamana_build_conf);
+
 
         auto diskann_ip = std::make_unique<knowhere::IndexDiskANN<float>>(
             kIpIndexDir + "/diskann", knowhere::metric::IP, std::make_unique<knowhere::LocalFileManager>());
         diskann_ip->BuildAll(nullptr, cfg);
         auto diskann_l2 = std::make_unique<knowhere::IndexDiskANN<float>>(
             kL2IndexDir + "/diskann", knowhere::metric::L2, std::make_unique<knowhere::LocalFileManager>());
         diskann_l2->BuildAll(nullptr, cfg);
+        auto merged_diskann_ip = std::make_unique<knowhere::IndexDiskANN<float>>(
+            kMergedIpIndexDir + "/diskann", knowhere::metric::IP, std::make_unique<knowhere::LocalFileManager>());
+        merged_diskann_ip->BuildAll(nullptr, merged_cfg);
+        auto merged_diskann_l2 = std::make_unique<knowhere::IndexDiskANN<float>>(
+            kMergedL2IndexDir + "/diskann", knowhere::metric::L2, std::make_unique<knowhere::LocalFileManager>());
+        merged_diskann_l2->BuildAll(nullptr, merged_cfg);
```

Comment:
suggest to move merge build related thing to the test function. Since this static setup is common for most of the test cases, but merge build is only for merge test case